queries.py

```diff
@@ -520,7 +520,8 @@ def is_webhook_valid(url):
         return False
 
     slack = "https://hooks.slack.com"
-    if (slack in url):
+    google_chat = "https://chat.googleapis.com/"
```

Comment:
nitpick:
`google_chat` this could have a better name? `base_google_chat_url` or simlar.